test/application_system_test_case.rb

```diff
@@ -1,5 +1,11 @@
 require "test_helper"
 
 class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
-  driven_by :selenium, using: :chrome, screen_size: [1400, 1400]
+  DRIVER = if ENV["DRIVER"]
+    ENV["DRIVER"].to_sym
```

Comment:
We might want have a set of permitted drivers son you can pass an invalid driver name.

Reply:
I have some options, but I want to check them before resolve

Reply:
I think that the description you added is enough for now